File: mq/rabbitmq/php/src/helloworld/producer.php
<?php
/**
 * rabbitmq hello-world
 */
require_once __DIR__ . '/../bootstrap.php';


use PhpAmqpLib\Connection\AMQPStreamConnection;
use PhpAmqpLib\Message\AMQPMessage;


$connection = new AMQPStreamConnection(RABBITMQ_MASTER, 5672, 'guest', 'guest');
$channel = $connection->channel();

$channel->queue_declare('hello', false, false, false, false);
while (true) {
    $channel->basic_publish(new AMQPMessage('hello world from php'), '', 'hello');
}

$channel->close();
/** @noinspection PhpUnhandledExceptionInspection */
$connection->close();

File: mq/rabbitmq/py/helloworld/producer.py
#!/usr/bin/env python3
#
# Copyright (C) 2019 jayson
#
from utils.connection import get_queue_connection
from config import RABBITMQ_MASTER


if __name__ == '__main__':
    with get_queue_connection(RABBITMQ_MASTER, 'hello') as channel:
        while True:
            channel.basic_publish(exchange='', routing_key='hello', body=b'hello world from python')
